news/2023/2023-12-22-how-advances-in-technology-affected-the-osu-community.md

```diff
@@ -0,0 +1,103 @@
+
+### Kuvster (2015)
+
+In 2015, Kuvster had a unique opportunity to snag the pp record under a very specific set of circumstances. Back in the day, ["TAG4" maps](/wiki/Beatmap/TAG) were [approved](/wiki/Beatmap/Category#approved), hence you were able to gain pp by playing them. They weren't really supposed to be played by in a single mode, but if you owned a touch screen and were apt with it, you were eligible to grab the pp record, which is exactly what they did:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/jcHiuNcwsiM" frameborder="0" allowfullscreen></iframe>
+</div>
+
+The 577pp record was short-lived though, as after this all "TAG4" beat maps were exempt from giving pp, and in 2019 they were moved to the [Loved category](/wiki/Beatmap/Category#loved). This however is not the event that "killed" the touch screen meta, its demise would come only 2 years later.
+
+### Freedomdiver (2017)
+
+Freedomdiver joined the osu! community in May 2016, and started as your average player, up until the moment they realised they can play osu! with the touch screen on their laptop.
+
+They quickly surged through the ranks after realizing that gaining pp by playing jump maps with a touch screen was simply easier given their point-to-point aim skill requirements. That however, did not just slip through without gathering negative attention from the community. The player base had turned against them, using them as a prime example of why touch screen was just unfair due to reasons stated before. However, it wouldn't be until November 2017 when freedomdiver decided to push touch screen to its limits, siding with the community and using actions instead of words to call for a change.
+
+The community somewhat sarcastically rallied around freedomdiver during one night in November, as they proceeded to break the pp record 3 times, all on the infamous [Yuima-ru*World TVver.](https://osu.ppy.sh/beatmapsets/459149#osu/994495) map. Hundreds of players spectated freedomdiver as they broke the pp record, and thanks to [CPUGeek](https://osu.ppy.sh/users/4172839) you can experience a snippet of the atmosphere surrounding the play:
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/JCU2ZWRj5gg" frameborder="0" allowfullscreen></iframe>
+</div>
+
+A month later, on December 15th, touch screen scores had their pp reduced along with an introduction of the brand new "Touch Device" mod, which would reduce the amount of pp gained from jumps on any map when osu! detected touch screen being used as a gameplay device.
+
+Nowadays, a new, unique usage for touch screen was figured. Some players use it exclusively as an aiming device, while still tapping to the beat with their good ol' keyboards. You can look at [bonk](https://osu.ppy.sh/users/4209965) and their [first person live play](https://www.youtube.com/watch?v=PE91EAzRcUo) to see what it looks like.
```

Comment:
there's no way this is a recent discovery. iirc some old cintiq players played this way

```suggestion
In the end, tapping and aiming with two hands isn't the only way to use a touch screen. Some players use them exclusively as aiming devices, while still tapping to the beat with their good ol' keyboards. You can look at [bonk](https://osu.ppy.sh/users/4209965) and their [first-person live play](https://www.youtube.com/watch?v=PE91EAzRcUo) to see what it looks like.
```

Reply:
yeah probably isnt new